src/main/java/codersafterdark/reskillable/base/configs/ConfigHandler.java

```diff
@@ -1,52 +1,56 @@
 package codersafterdark.reskillable.base.configs;
 
+import codersafterdark.reskillable.Reskillable;
 import codersafterdark.reskillable.base.LevelLockHandler;
+import codersafterdark.reskillable.base.configs.json.LockJson;
+import codersafterdark.reskillable.base.configs.json.LockTypeJsonFactory;
 import codersafterdark.reskillable.lib.LibMisc;
-import com.google.gson.Gson;
-import com.google.gson.GsonBuilder;
+import com.google.gson.JsonParseException;
+import com.google.gson.reflect.TypeToken;
 import net.minecraftforge.common.MinecraftForge;
 import net.minecraftforge.common.config.Configuration;
 
 import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 
 import static codersafterdark.reskillable.base.configs.ConfigUtilities.loadPropBool;
 
 public class ConfigHandler {
 
-    /////////////////
-    // Directories //
-    /////////////////
-    private static File configDir;
-    private static File jsonDir;
-
-
     /////////////
     // Configs //
     /////////////
     public static Configuration mainConfig;
-    public static Gson locks;
-
-    ////////////////////
-    // Default Values //
-    ////////////////////
 
     /// Main Config ///
     public static boolean disableSheepWool = true;
     public static boolean enforceFakePlayers = true;
+
+    ////////////////////
+    // Default Values //
+    ////////////////////
     public static boolean enableTabs = true;
     public static boolean enableLevelUp = true;
     public static boolean hideRequirements = true;
-
     public static Map<String, Configuration> cachedConfigs = new HashMap<>();
+    /////////////////
+    // Directories //
+    /////////////////
+    private static File configDir;
+    private static File jsonDir;
 
     public static void init(File file) {
         generateFolder(file);
         mainConfig = new Configuration(new File(configDir.getPath(), "reskillable.cfg"));
         mainConfig.load();
         loadData();
-        loadJSONLocks();
+        // loadJSONLocks();
```

Comment:
Can remove this line as it is being loaded from `LevelLockHandler#setupLocks` and there is no reason to have this here just commented out.